actions/post_fit.py

```diff
@@ -18,7 +18,7 @@
 # __maintainer__ = "Rick Kauffman"
 # __email__ = "rick.a.kauffman@hpe.com"
 
-# A python script for sending a 'fit' to the cfm controller
```

Comment:
Did you mean to change this description? Seems like you might have changed the wrong file. Post_fit.py should be sending a fit to the controller, not add vlans, no?